app/main.go

```diff
@@ -74,9 +77,19 @@ func mainEx(argv []string) {
 	processor, err := freki.New(args["--interface"].(string), rules, logger)
 	onErrorExit(err)
 
+	// TODO: this all needs to move _inside_ freki
 	processor.AddServer(freki.NewTCPLogger(tcpLoggerPort, 1024))
 	processor.AddServer(freki.NewHTTPLogger(httpLoggerPort))
 	processor.AddServer(freki.NewTCPProxy(tcpProxyPort))
+	processor.AddServer(freki.NewUserConnServer(userConnServerPort))
+
+	processor.RegisterConnHandler("echo", func(conn net.Conn, md *freki.Metadata) error {
```

Comment:
@glaslos can you show an example of this? The handler is a _TCP_ echo implementation, not ICMP.